domain/model/state/modelstate.go

```diff
@@ -147,7 +148,10 @@ func (s *ModelState) Model(ctx context.Context) (coremodel.ReadOnlyModel, error)
 	if owner := m.CredentialOwner; owner != "" {
 		model.CredentialOwner, err = user.NewName(owner)
 		if err != nil {
-			return coremodel.ReadOnlyModel{}, errors.Trace(err)
+			return coremodel.ReadOnlyModel{}, errors.Errorf(
+				"getting model read only information and parsing model owner username %q: %w",
```

Comment:
We don't need `getting model read only information and` in every error message. The caller of the method can add that annotation if it wants to.

Reply:
Fixed up.